src/backend/legendary/games.ts

```diff
@@ -502,6 +505,24 @@ class LegendaryGame extends Game {
       return { status: 'error', error: res.error }
     }
     this.addShortcuts()
+
+    const [anticheatInfo, setAnticheatInfo] = useState<AntiCheatInfo | null>(
```

Comment:
this is a react feature, it can't be used like this in the backend that is not a react app